### ISSUE TITLE
aarch64: does not build on Ubuntu 20.04/10

### ISSUE THREAD

**Issue by hlef** [cloudius-systems/osv#1125]
Following [the wiki](https://github.com/cloudius-systems/osv/wiki/AArch64), `./scripts/build` fails with the following output:
```
# ./scripts/build image=empty fs=ramfs -j4 arch=aarch64
Building into build/release.aarch64
Makefile:203: *** Missing build/downloaded_packages/aarch64/gcc/install directory. Please run "./scripts/download_aarch64_packages.py".  Stop.
make failed. Exiting from build script
```
Running `./scripts/download_aarch64_packages.py` does not help; on Ubuntu this script only seems to install libboost related files.

Am I doing something wrong? This happens on both 20.04 and 20.10.

**Comment by wkozaczuk** [comment]
Did you run scripts/setup.py? For Ubuntu, it should install all
cross-compile packages except boost. It works on my Ubuntu.

Waldek

**Comment by hlef** [comment]
Hi Waldek, thanks for the quick answer!

Indeed, I forgot to run `scripts/setup.py`. Running it did the trick and the compilation succeeded. I recommend editing this error message to suggest the user to run `setup.py`.

Oh, and, another nit in case you are interested:
```
./scripts/run.py --arch=aarch64 -e '/tests/tst-hello.so'
'qemu-system-aarch64' binary not found. Please install the qemu-system-x86 package.
```

This should probably be `s/qemu-system-x86/qemu-system-aarch64/` :)

**Comment by wkozaczuk** [comment]
@hlef We have very few volunteers, so would you mind sending a patch fixing the `s/qemu-system-x86/qemu-system-aarch64/` problem?

**Comment by hlef** [comment]
Sure, let's leave this bug report open as a reminder.

**Comment by wkozaczuk** [comment]
Fixed by 4d329346c02e4289e480497addd429fea5b5e474
